templates/registration/password_reset.html

```diff
@@ -0,0 +1,21 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+{% load account %}
+
+{% block head_title %}{% trans "Reset Password" %}{% endblock %}
+
+
+{% block content %}
+<h1>{% trans "New Password" %}</h1>
+<P>{% trans 'Write and repeat your new password.' %}</P>
+<form method="post" action="{% url 'rest_password_reset_confirm' %}">
```

Comment:
Does the `rest_password_reset_confirm` redirect to a success page or something of the sort?